CosmosApiChangeFeedForMessageRetention/function.json

```diff
@@ -0,0 +1,17 @@
+{
+  "bindings": [
+    {
+      "type": "cosmosDBTrigger",
+      "name": "documents",
+      "direction": "in",
+      "leaseCollectionName": "%LEASE_MESSAGE_STATUS%",
+      "leaseCollectionPrefix": "%LEASE_COLLECTION_PREFIX%",
+      "connectionStringSetting": "COSMOSDB_CONNECTION_STRING",
+      "databaseName": "%COSMOSDB_NAME%",
+      "collectionName": "%COSMOSDB_MESSAGE_STATUS_CONTAINER_NAME%",
+      "createLeaseCollectionIfNotExists": true,
+      "startFromBeginning": true
+    }
+  ],
+  "scriptFile": "../dist/CosmosApiChangeFeedForMessageRetention/index.js"
```

Comment:
We should add a `retry` block, like [here](https://github.com/pagopa/io-functions-elt/blob/5489883d73dd412363169903052a00127c5b3ec9/AnalyticsMessagesChangeFeedInboundProcessorAdapter/function.json#L17)

This is needed to make framework retry the tasks in case an error is throwed.